package.json

```diff
@@ -6,13 +6,13 @@
     "changeset:release": "pnpm build && changeset publish",
     "changeset:version": "changeset version && pnpm install --lockfile-only",
     "dev": "pnpm --parallel --filter './packages/**' --filter './references/packages/**' dev",
-    "devPreinstall": "git submodule init && git submodule update && npx only-allow pnpm",
     "docs:build": "pnpm --parallel --filter './packages/**' --filter docs build",
     "docs:dev": "pnpm dev && pnpm --filter docs dev",
     "lint": "eslint . --cache",
     "lint:fix": "pnpm lint --fix",
     "lint:format": "prettier --write",
     "playground": "pnpm dev && pnpm --filter example-dev dev",
+    "preinstall": "git submodule init && git submodule update && pnpm i --filter './references/packages/**' --ignore-scripts && npx only-allow pnpm",
```

Comment:
This does a few things:
- `git submodule init && git submodule update` — pull in `@wagmi/references` submodule.
- `pnpm i --filter './references/packages/**' --ignore-scripts` — install deps for only `@wagmi/references` and skip any other scripts (e.g. `postinstall`). this is required since the `pnpm i` command that triggered `preinstall` was already issued without references since it didn't exist at the time.
- `npx only-allow pnpm` — only allow pnpm to be used for package management.